Acornfile

```diff
@@ -0,0 +1,51 @@
+name:        "Rust Sample Acorn"
+description: "Acorn running a sample Rust app"
+readme:      "./README.md"
+
+
+args: {
+	// Name of the database to create. Defaults to "instance"
+	dbName: "rust-sqlx"
+}
+
+services: db: {
+    image: "ghcr.io/acorn-io/postgres:v15.#-#"
+    serviceArgs: {
+        dbName: args.dbName
+    }
+}
+
+containers: {
+  backend: {
+    image: "ghcr.io/infracloudio/rust-backend:v1"
+    env: {
+    "POSTGRES_HOST": "@{service.db.address}"
+    "POSTGRES_PORT": "@{service.db.port.5432}"
+    "POSTGRES_USER": "@{service.db.secrets.admin.username}"
+    "POSTGRES_PASSWORD":"@{service.db.secrets.admin.password}"
+    "POSTGRES_DB":"@{service.db.data.dbName}"
+    "DATABASE_URL":"postgresql://@{service.db.secrets.admin.username}:@{service.db.secrets.admin.password}@@{service.db.address}:5432?schema=public"
+    "PGADMIN_DEFAULT_EMAIL":"admin@admin.com"
+    "PGADMIN_DEFAULT_PASSWORD":"password123"
+    }
+    ports: {
+      publish: "8000:8000/http"
+    }
+    memory: 1048Mi
+    cmd: ["/bin/sh", "-c", "cd /usr/src/fullstack-rust-app/backend && sqlx migrate run; cargo run --bin backend -- --port 8000"]
```

Comment:
@aashimodi14 Why Any particular reason to put the command here instead of the dockerfile?

Reply:
yes, we need to perform sql migrations once postgres is up, so if the migrations are not available during build (crago build command), rust fails to compile/build.

Reply:
But can't this command be moved inside the dockerfile? We should only be overriding the command here for special cases, instead of skipping the command completely in the dockerfile.

Reply:
updated the dockerfile